myapp/spec/system/tasks_spec.rb

```diff
@@ -20,14 +17,26 @@
 
     context 'delete a task' do
       it "delete the task with button" do 
+        task = Task.create!(title: 'Test title 1', description: 'Test Description 1')
+
         visit tasks_path
-        click_on @task1.id.to_s
+        click_on task.id.to_s
 
         expect(page).to have_no_content('Task Detail 1')
         expect(Task.all.length).to eq(1)
         expect(page).to have_content('Deleted task successfully!')
       end
     end
+
+    it 'show tasks in due date order' do 
+      task1 = Task.create!(title: 'Test title 1', description: 'Test Description 1', due: '2024-04-01 20:00:00')
```

Comment:
Just one thing!
We can delete this variables, since it is not used!

Reply:
Removed them. Thank you! 
64b0afe